scripts/redux/actions.js

```diff
@@ -224,6 +224,16 @@ const subscribeActions = {
     })
     .catch(() => {
        store.dispatch({
+        type: SET_DIALOG_DATA,
+        dialog: {
+          ['subscribe']: {
+            isOpened: true,
+            data: Object.assign(data, { errorOcurred: true })
```

Comment:
in this case you modify `data` instead of creating a new object. Do `Object.assign({}, data, { errorOcurred: true })`